types/V1.mli

```diff
@@ -482,7 +482,8 @@ module type ARP = sig
   val set_ips : t -> ipaddr list -> unit io
 
   (** [remove_ip arp ip] removes [ip] to the bound IP address list in
-      the [arp] value, which will xmit a GARP packet also. *)
+      the [arp] value, which will xmit a GARP packet for any remaining IPs in
```

Comment:
It's a shorthand (see http://painintheenglish.com/case/1053 ), and I don't think saving ~20 bytes is worth even half a second of human confusion.